### DISCUSSION TITLE
Private Retrieval

### DISCUSSION THREAD

**Discussion by KarolaKirsanow** [protocol/research#63] [Ideas, Open Problems, and Proposals]
Discussions related to private Data retrieval Open Problems

**Comment by agowa** [comment]
The [blog post](https://research.protocol.ai/blog/2022/new-open-problems-in-private-data-retrieval/) is linked to this discussion thread, but it looks vacant. So is this the right place or have discussions moved elsewhere?
And is this the associated problem description? https://github.com/protocol/ResNetLab/blob/master/OPEN_PROBLEMS/PRESERVE_USER_PRIVACY.md

**Reply by willscott** [reply]
* This is the right place! Welcome!
* the blog post is going to be updated - there isn't yet a place to submit proposals yet, so expect a bit more fleshing out.
* The problem you link is an older revision. the one associated with this blog post / discussion is https://github.com/protocol/research#private-retrieval

**Reply by jsoares** [reply]
The RFP and application system are now up:
https://research.protocol.ai/blog/2022/announcing-rfp-014-the-one-with-private-retrieval/
